Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/AppContentView.swift

```diff
@@ -44,6 +44,7 @@ struct AppContentView: View {
             }
             #endif
         }
+        .handleRevenueCatDeeplinks()
```

Comment:
Nothing to do here but I must say, I'm still not used to having these things as modifiers of a view, seems like a very weird API to me 😅

Reply:
🤔 what would a more natural API look like to you as a dev? I'm open to anything as long as its easy to do. I know others do the setup in the app delegate, but the industry seems to be moving away from that

Reply:
Yeah, I think this is the way it's supposed to be in SwiftUI (like the `onOpenURL` API we're using). Just seems very weird to tie this behavior to a view IMO... So yeah, I think this is the most SwiftUI solution for now.

Reply:
Yes I feel you Toni haha, but indeed this seems to be the SwiftUI way. In Compose it would be more natural to add a route to a navigation graph. The downside of that is that there are multiple ways to build such a graph, as mentioned on Slack.